src/models/profile-keys.ts

```diff
@@ -0,0 +1 @@
+export const ProfileKeyMSC4175Timezone = "us.cloke.msc4175.tz";
```

Comment:
This is used in https://github.com/matrix-org/matrix-react-sdk/pull/12967 and made slightly more sense to expose as a const here, than as part of Element. Thoughts?

Reply:
Yeah, I'm always unsure about these things but it feels useful to have here I guess.

Reply:
I'll add a docstring, because I think that would probably be a kind thing to do.